hash/apple2_flop_clcracked.xml

```diff
@@ -45860,6 +43874,144 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="kquest">
+		<description>King's Quest: Quest for the Crown (4am and san inc crack)</description>
+		<year>1984</year>
+		<publisher>Sierra On-Line</publisher>
+		<info name="programmer" value="Roberta Williams, Jeff Stephenson, Arthur Abraham, Charles Tingley, Ken MacNeill, Doug MacNeill, and Greg Rowland" />
+		<info name="usage" value="Requires a 128K Apple //e or later." />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!-- Dump released: 2018-01-15 -->
+		<!-- adventure game -->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="king's quest (4am and san inc crack) disk a.dsk" size="143360" crc="b7542a8a" sha1="57df97d42cb6ba3e08a725823de2ba0f00b799a0" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="king's quest (4am and san inc crack) disk b.dsk" size="143360" crc="b39dc9ff" sha1="612d93007050cd47f1216b406a7fbb62f9ab75a0" />
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="king's quest (4am and san inc crack) disk c.dsk" size="143360" crc="2fd0b2b2" sha1="2c179d77b3216af2d30dc083369052054258b5a9" />
+			</dataarea>
+		</part>
```

Comment:
Should this one have `part_id` for the disk images, or do we lack reliable labels?  Does the game use names for the disks on-screen that can be used even if we don’t have photos?